src/components/TripInfo.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from "react";
+import TextareaAutosize from "@mui/base/TextareaAutosize";
+import {
+  TextField,
+  Select,
+  MenuItem,
+  FormControl,
+  InputLabel,
+} from "@mui/material";
+import style from "./TripInfo.module.css";
+
+function TripInfo(props: any) {
```

Comment:
Can you try to define Type/interface for props? We should try to avoid using any when working with TS.